frontend/nc-games/src/components/SingleReview.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { getReview } from "../api";
+import { useParams } from "react-router-dom";
+
+export default function SingleReview() {
+    const {review_id} = useParams()
+    const [review, setReview] = useState({});
+    const [loading, setLoading] = useState(true);
+
+    useEffect(()=>{ 
+        getReview(review_id)
+        .then((reviewFromApi) => {
+            
+            if(review_id === undefined){   
+                
+                setReview(<h1>where do you think you're going</h1>);
+                setLoading(false);
```

Comment:
this if statement here is a little redundant as if review_id is undefined the api call will fail with a 400 so this `<h1>` will never render. This could be added into a catch block at a later time but it's not essential now